src/engine/Graphics/Context/texture-loader.ts

```diff
@@ -152,10 +157,12 @@ export class TextureLoader {
       return null;
     }
 
-    let tex: WebGLTexture = null;
     if (this.has(image)) {
-      tex = this.get(image);
-      gl.deleteTexture(tex);
+      const tex = this.get(image);
```

Comment:
```suggestion
      const texture = this.get(image);
```

It's an existing variable name, but the boyscout it me wants to leave the code better than it was